README.md

```diff
@@ -0,0 +1,21 @@
+# Industrial Taxonomy
```

Comment:
Shall we add an issue for creating a front page description of the project? Maybe with a nice image